insights/parsers/ps.py

```diff
@@ -104,22 +105,76 @@ def users(self, proc):
         for row in self.data:
             if proc == row["COMMAND"]:
                 if row["USER"] not in ret:
-                    ret.update({row["USER"]: []})
+                    ret[row["USER"]] = []
                 ret[row["USER"]].append(row["PID"])
         return ret
 
     def fuzzy_match(self, proc):
-        """boolean: Returns ``True`` if ``proc`` is in the COMMAND column."""
-        return proc in "".join(self.running)
+        """boolean: Returns ``True`` if the word ``proc`` appears in the COMMAND column."""
+        return any(proc in row['COMMAND'] for row in self.data)
 
     def __contains__(self, proc):
-        return proc in self.running
+        # We should be able to take this if out when we only have the PsAuxww
+        # parser:
+        return any(proc == row['COMMAND'] for row in self.data if 'COMMAND' in row)
```

Comment:
Is this change necessary?

Reply:
Yeah, probably not - sorry, got a bit carried away.